src/core/endpoint.ml

```diff
@@ -9,7 +9,7 @@ let error_msg str = Error (`Msg str)
 let error_msgf fmt = Format.kasprintf error_msg fmt
 
 let of_string str =
-  let ( >>= ) = Result.bind in
+  let ( >>= ) = Stdlib.Result.bind in
```

Comment:
I think just bump up the minimum OCaml version. This still fails on 4.07

Reply:
Hmmhmm, it's a bit of shame to bump the version because we don't have `Result.bind` ... Re-write `bind` is fine for me to support more OCaml versions.